docs/agent_api.rst

```diff
@@ -3,4 +3,313 @@
+
+Removes a DDS participant from a certain client proxy.
+Returns `true` if the participant was deleted, or `false` otherwise.
+
+:client_key: The identifier of the `ProxyClient` from which the participant must be deleted.
+:participant_id: The ID of the participant to be deleted.
+:op_result: The result status of the operation.
+
+------
+
+.. code-block:: cpp
+
+    bool create_<entity>_by_xml(uint32_t client_key, uint16_t <entity>_id, uint16_t <associated_entity>_id, const char* xml, uint8_t flag, OpResult& op_result);
+
+Creates a certain DDS entity attached to an existing `ProxyClient`, given its client key.
+
+An XML must be provided, containing the DDS description of the entity to be created.
+
+There are as many methods available as existing DDS entities, replacing the parameters *<entity>* and *<associated_entity>* as follows:
+
+.. _existing_entities_and_associated_entities:
+
+*Agent's API available DDS entities and their associated entities*
+******************************************************************
+
+=============== =========================
+**<entity>**    **<associated_entity>**
+=============== =========================
+topic           participant
+publisher       participant
+subscriber      participant
+datawriter      publisher
+datareader      subscriber
+requester       participant
+replier         participant
+=============== =========================
+
+This operation returns `true` if the entity is successfully created and linked to its associated entity (which must previously exist in the given `ProxyClient`), or false otherwise.
+
+:client_key: The identifier of the `ProxyClient` to which the resulting entity will be attached to.
+:<entity>_id: The ID of the DDS entity to be created.
+:<associated_entity>_id: The identifier of the DDS entity to which this one will be assocciated.
+:xml: The XML describing the entity properties.
+:flag: It determines the creation mode of the new entity (see :ref:`creation_mode_table`).
+:op_result: The result status of this operation.
+
+------
+
+.. code-block:: cpp
+
+    bool create_<entity>_by_ref(uint32_t client_key, uint16_t <entity>_id, uint16_t <associated_entity>_id, const char* ref, uint8_t flag, OpResult& op_result);
+
+Creates a certain DDS entity attached to an existing `ProxyClient`, given its client key.
+
+The description of the entity to be created is hosted in a certain references file and must be tagged with the same tag name, provided as the `ref` parameter to this method.
+
+There are as many methods available as existing DDS entities, replacing the parameters *<entity>* and *<associated_entity>* as shown above in the previous method description (see :ref:`existing_entities_and_associated_entities`).
+
+This operation returns `true` if the entity is successfully created and linked to its associated entity (which must previously exist in the given `ProxyClient`), or false otherwise.
```

Comment:
```suggestion
This operation returns ``true`` if the entity is successfully created and linked to its associated entity (which must previously exist in the given ``ProxyClient``), ``false`` otherwise.
```